src/components/status/index.tsx

```diff
@@ -84,12 +102,180 @@ const Status: React.FC = () => {
     }
   };
 
+  const fetchDR = async () => {
+    // get bearer token
+    const user = JSON.parse(
+      localStorage.getItem('BEMS_USER') ||
+        sessionStorage.getItem('BEMS_USER') ||
+        '{}',
+    );
+    // GET to User Info API
+    const response = await fetch(
+      `${process.env.REACT_APP_BACKEND_ENDPOINT}/DR_result?start_date=2021-01-01&end_date=2021-01-02`,
```

Comment:
Please change the data range to 2021/01/01~today+2

Reply:
done